### PR TITLE
Add search function for Config Editor

### PR DESCRIPTION
**Pull request by ray007330**
## Description

We add search button on the user interface to enhance the usability.
image
It would be helpful when the number of knobs is large.

For details on how to complete these options and their meaning refer to [CONTRIBUTING.md](https://github.com/microsoft/mu/blob/HEAD/CONTRIBUTING.md).

- [ ] Impacts functionality?
- [ ] Impacts security?
- [ ] Breaking change?
- [ ] Includes tests?
- [ ] Includes documentation?

## How This Was Tested

Open a config xml file and then search for the key words you want

## Integration Instructions

None

### CODE REVIEW

**Review by os-d — APPROVED** [review]

**Review by kuqin12 — APPROVED** [review]

File: SetupDataPkg/Tools/ConfigEditor.py [hunk 1]
```diff
@@ -616,6 +645,169 @@ def __init__(self, master=None):
...
+        self.current_matches = []
+        self.current_match_index = -1
+
+        def expand_tree_to_page(page_id_list):
+            level_3_list = []
+            for level_1 in self.left.get_children():
+                for level_2 in self.left.get_children(level_1):
+                    for level_3 in self.left.get_children(level_2):
+                        level_3_list.append(level_3)
+            print(f'level3: {level_3_list}')
```
Comment (apop5) [inline SetupDataPkg/Tools/ConfigEditor.py:658]: These prints appear to be left over from debugging, correct? If so, they can be removed.

File: SetupDataPkg/Tools/ConfigEditor.py [hunk 2]
```diff
@@ -616,6 +645,169 @@ def __init__(self, master=None):
...
+            for level_1 in self.left.get_children():
+                for level_2 in self.left.get_children(level_1):
+                    for level_3 in self.left.get_children(level_2):
+                        for page_id in page_id_list:
+                            if page_id == level_3:
+                                self.left.item(level_3, tags="highlight")
+                                self.left.see(level_3)
+
+        def search_in_config_data(search_term):
+            print("-----------------------cfg_data_list content-----------------------")
```
Comment (apop5) [inline SetupDataPkg/Tools/ConfigEditor.py:682]: Same for the prints here

**Comment by apop5** [comment]
This looks fine, it just has a lot of print messages coming out that are debugging type information.

**Review by apop5 — APPROVED** [review]

**Comment by ray007330** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by ray007330** [comment]
@os-d @apop5 @kuqin12 

I refactored the code and removed some unused print statement.
Do you have time take a look again?

**Review by os-d — APPROVED** [review]

**Pull request merged**